core/src/test/java/org/elasticsearch/search/aggregations/metrics/cardinality/InternalCardinalityTests.java

```diff
@@ -38,13 +37,21 @@
     private static List<HyperLogLogPlusPlus> algos;
     private static int p;
 
-    @Before
-    public void setup() {
+    @After //we force @After to have it run before ESTestCase#after otherwise it fails
+    @Override
+    public void setUp() {
```

Comment:
yes it does. I have messed these two up for some reason. Probably ran only AggregationsTests and not the cardinality tests...fixing